### ISSUE TITLE
SteamAPI_InitSafe vs SteamAPI_Init?

### ISSUE THREAD

**Issue by keithjohnston** [aeb-dev/steamworks_gen#1]
For Mac using steamworks v1.58a 26th October 2023, I had to change SteamAPI_Init to SteamAPI_InitSafe here:

steam_api.dart:
```
final _init =
    dl.lookupFunction<Bool Function(), bool Function()>("SteamAPI_InitSafe");
```

**Comment by aeb-dev** [comment]
There is no such function listed anywhere how did you find it?

**Comment by keithjohnston** [comment]
I had to dump out all the symbols in the dynamic library for the Mac.

**Comment by aeb-dev** [comment]
I see. Well I don't think that is the appropriate way of initializing. I feel like something else is wrong and you are side-stepping some check in order to get things "seem" running.

The reason why I think like that is steam already distributes public api list in a json file. `SteamAPI_InitSafe` does not exist there so I assume you should not be using that function

**Comment by aeb-dev** [comment]
Released a new version which uses new api of 1.59 `InitFlat`
